utils_cv/detection/bbox.py

```diff
@@ -145,6 +216,64 @@ def from_array(cls, arr: List[int], **kwargs) -> "AnnotationBbox":
         bbox.set_meta(**kwargs)
         return bbox
 
+    @classmethod
+    def from_array_xywh(cls, arr: List[int], **kwargs) -> "AnnotationBbox":
```

Comment:
I don't feel good about the **kwargs, and generally about so many new helper functions. Can you please remove all functions which are not used and I will then look at them again.

Reply:
All removed.  Those functions are legacy code which I forgot to remove, and I also added tests for them :man_facepalming: